libbeat/processors/add_kubernetes_metadata/docs/add_kubernetes_metadata.asciidoc

```diff
@@ -89,4 +89,4 @@ client. It defaults to `KUBECONFIG` environment variable if present.
 `default_indexers.enabled`:: (Optional) Enable/Disable default pod indexers, in
 case you want to specify your own.
 `default_matchers.enabled`:: (Optional) Enable/Disable default pod matchers, in
-case you want to specify your own.
+case you want to specify your own.
```

Comment:
+1 on @mtojek 's comment regarding the usage `you`s . This does not sound/look so good to, I would prefer a passive voice. 

But anyway this is a general issue of this doc and not for this patch specifically. It would be nice to have it changed here since we are on it if we don't have a plan on revisiting the docs in general.